doc/attr.xml

```diff
@@ -133,6 +133,30 @@ gap> DigraphNrEdges(D);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DigraphNrLoops">
```

Comment:
This chunk shouldn't be here, looks like you've branched `DotDigraphColours` off `DigraphNrLoops` but you should have done it from the `master` branch.